packages/schema/src/core/models/BaseField.ts

```
@@ -0,0 +1,337 @@
+    this.description = description;
+  };
+
+  setDisplay = (type?: FieldDisplayTypes) => {
+    this.display = type;
+  };
+
+  setPattern = (type?: FieldPatternTypes) => {
+    this.pattern = type;
+  };
+
+  setComponent = <C extends JSXComponent, ComponentProps extends object = {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)